ctapipe/io/tests/test_zfits.py

```diff
@@ -0,0 +1,33 @@
+from pkg_resources import resource_filename
+import os
+
+example_file_path = resource_filename(
```

Comment:
Is it possible to use `from ctapipe.utils import get_dataset` instead?

Reply:
I think not. The file which is needed for this test is not in one of the places, where this function is searching I presume. Unless I misunderstood that function.